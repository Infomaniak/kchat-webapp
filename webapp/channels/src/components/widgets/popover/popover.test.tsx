// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import {render} from '@testing-library/react';
import React from 'react';

import Popover from '.';

describe('components/widgets/popover', () => {
    test('plain', () => {
        const {container} = render(
            <Popover id='test'>
                {'Some text'}
            </Popover>,
        );
        expect(container.firstChild).toMatchSnapshot();
    });
});
