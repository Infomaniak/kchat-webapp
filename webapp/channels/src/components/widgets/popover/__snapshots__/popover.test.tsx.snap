// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/widgets/popover plain 1`] = `
<div
  class="popover popover-sm popover-info right"
  id="test"
  role="tooltip"
  style="display: block;"
>
  <div
    class="arrow"
  />
  <div
    class="popover-content"
  >
    Some text
  </div>
</div>
`;
